packages/store/src/devicesSlice/selectors.ts

```diff
@@ -77,8 +77,14 @@ export const getSelectedGroupInfo = createSelector(
   }
 );
 
-export const getLimitMaxed = createSelector([getAcceptedDevices, getDeviceLimit], ({ total: acceptedDevices = 0 }, deviceLimit) =>
-  Boolean(deviceLimit && deviceLimit <= acceptedDevices)
+export const getCombinedLimit = createSelector(
+  [getDeviceLimits],
+  deviceLimits => Object.values(deviceLimits).reduce((accu, limit) => accu + Math.max(limit ?? 0, 0), 0) // limit might be -1 for unlimited devices
+);
+
+export const getLimitMaxed = createSelector(
```

Comment:
I think we might need more sophisticated getLimitMaxed detector in the future, per tier one